Documentation/mender-ci-cd.md

```diff
@@ -0,0 +1,105 @@
+# Mender CI/CD
+
+This document contains various guides and how-to's for topics around our current
+CI/CD setup using GitLab on Google Cloud Platform (GCP). For a basic introduction
+to GitLab, refer to the dedicated guide in `gitlab-ci.md`.
+
+
+## Enable KVM acceleration on GCP
+Enabling KVM on Google Cloud's Compute Engine there are two steps:
+1. Create a KVM-enabled custom image using the following license URL: `https://compute.googleapis.com/compute/v1/projects/vm-options/global/licenses/enable-vmx`
+2. Install and enable kvm module for qemu
+
+For step one, assuming you have [google-cloud-sdk](https://cloud.google.com/sdk) 
+installed and initialized to current GCP project, the following command will 
+create a new `Ubuntu 18.04` image with kvm enabled:
+```shell
+gcloud compute images create <image-name> \
+    --source-image-project=ubuntu-os-cloud \
+    --source-image-family=ubuntu-1804-lts \
+    --licenses="https://www.googleapis.com/compute/v1/projects/vm-options/global/licenses/enable-vmx"
+```
```

Comment:
This `image-name` is what then we will use in the Gitlab runner config file (autoscale part). Can we add a note for this?

Reply:
Done.